ci-operator/step-registry/single-node/conf/e2e/single-node-conf-e2e-commands.sh

```diff
@@ -19,3 +19,33 @@ spec:
     systemReserved:
       memory: 3Gi
 EOF
+
+# Use first four cores on a single node for workload partitioning
```

Comment:
A comment link here to the [enhancement](https://github.com/openshift/enhancements/blob/master/enhancements/workload-partitioning/management-workload-partitioning.md?rgh-link-date=2021-09-02T12%3A23%3A39Z#goals) would be useful for future reference